### PR TITLE
Update versions to latest

### PR DESCRIPTION
**Pull request by lorypelli**
Let me know if you want something to not be updated, but I think it's better to update everything...

### CODE REVIEW

**Comment by Jacalz** [comment]
Have you built the images locally and verified that they work?

**Comment by lorypelli** [comment]
> Have you built the images locally and verified that they work?

I'm having troubles make go running locally, I don't know why it doesn't want to set the PATH on linux

**Comment by lorypelli** [comment]
> Have you built the images locally and verified that they work?

I need to change sha256

**Comment by Jacalz** [comment]
You should not knowingly open a PR with untested changes. That's very bad practice

**Comment by lorypelli** [comment]
> You should not knowingly open a PR with untested changes. That's very bad practice. If something is work in progress, at least open as a draft

Now I managed to test

**Comment by lorypelli** [comment]
@Jacalz building works correctly

**Comment by Bluebugs** [comment]
I would prefer one pr for each change and with a descriptive change in each case (like 'updating fyne to v2.4.3')

File: base/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,9 @@
 FROM debian:bullseye-slim AS base
 
-ENV GO_VERSION=1.21.1
-# dev version of Zig to support windows-386 target
```
Comment (andydotxyz) [inline base/Dockerfile:4]: This looks like an important comment, please don't remove it

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Thanks for this, but watch you don't remove important content.

Has this combination of versions worked well for you?

**Comment by andydotxyz** [comment]
> I would prefer one pr for each change and with a descriptive change in each case (like 'updating fyne to v2.4.3')

Surely that is a lot of overhead for version bumps. If you really need it a separate commit for each managed project, but not a whole PR?

**Comment by lorypelli** [comment]
> Thanks for this, but watch you don't remove important content.
> 
> Has this combination of versions worked well for you?

I would like to update also zig version but I wasn't able to find sha256, so I restored it to dev, but I didn't add bacl the comment, I will try to find the sha but now I don't have time...

**Comment by lorypelli** [comment]
I added the comment back for now, you can merge it if you want it builds correctly

**Comment by lorypelli** [comment]
Actually, I wasn't able to run any command inside of the docker container, it wasn't giving output of them, idk why so I recommend you test this locally by yourself

**Comment by lorypelli** [comment]
I was able to find it

File: base/Dockerfile (hunk 1 above)
Comment (lorypelli) [reply base/Dockerfile:4]: I was able to update zig version also

**Comment by Bluebugs** [comment]
Updating zig is a high risk operation. Have you tried to build a few application with this new image using fyne-cross and tested them on their target environment.

**Comment by lorypelli** [comment]
> Updating zig is a high risk operation. Have you tried to build a few application with this new image using fyne-cross and tested them on their target environment.

If you think it can be an issue, let me know and I will revert the change and make another pull request when I will be able to make sure everything works with it

**Comment by lorypelli** [comment]
ok, for now I reverted it, let me know if you are able to do some tests with it

**Comment by Jacalz** [comment]
> > I would prefer one pr for each change and with a descriptive change in each case (like 'updating fyne to v2.4.3')
> 
> Surely that is a lot of overhead for version bumps. If you really need it a separate commit for each managed project, but not a whole PR?
> 

I agree. One commit for each would be more sensible.

**Comment by andydotxyz** [comment]
> @[lorypelli](https://github.com/lorypelli) lorypelli requested a review from [andydotxyz](https://github.com/andydotxyz) [3 days ago](https://github.com/fyne-io/fyne-cross-images/pull/41#event-11344807016)

I'm not the right person to review this - but others are around who know it well.

**Comment by Jacalz** [comment]
Sorry @lorypelli, I think this got superseded by https://github.com/fyne-io/fyne-cross-images/pull/43. Apologies.

**Comment by Jacalz** [comment]
I closed this now but you are more than welcome to open a new PR for that workflow misspelling. I don't think there is any need to update fixuid because the latest release of fyne-cross doesn't use it as far as I know.

**Pull request closed without merging**